tests/run_tests.py

```diff
@@ -138,11 +142,15 @@ def main():
     parser.add_argument('--vic_exe', type=str,
                         help='VIC executable to test',
                         default=os.path.join(
-                            test_dir, '../vic/drivers/classic/vic_classic.exe'))
+                            test_dir, '../vic/drivers/classic/vic_classic.exe'),
+                        nargs='+')
     parser.add_argument('--driver', type=str,
-                        help='VIC driver to test',
+                        help='VIC driver to test. Can be a list of drivers '
+                             'for cross-driver tests, in which case a list of '
+                             'corresponding vic_exe in the same order must '
+                             'also be specified',
                         choices=['classic', 'image'],
-                        default='classic')
+                        default='classic', nargs='+')
```

Comment:
maybe it would make more sense to combine the `vic_exe` and `driver` command line arguments such that we specify something like:

`run_tests.py --classic=../vic/drivers/classic/vic_classic.exe --image=/drivers/image/vic_image.exe ...`